node/forcedroid.js

```diff
@@ -300,7 +291,6 @@ function createNativeApp(config, showNextSteps) {
     // copy <Android Package>/libs/SalesforceSDK -> <App Folder>/forcedroid/libs/SalesforceSDK
     var salesforceSDKRelativePath = path.join('libs', 'SalesforceSDK');
     copyFromSDK(packageSdkRootDir, config.targetdir, salesforceSDKRelativePath);
```

Comment:
I looked into some solutions, including that one, but it's a bit intrusive. The ```Gradle``` team is working on a solution to this, and the next gen ```Android``` plugin is said to solve some of these problems too, so I figured we could wait a bit.